backend/app/utils/pdf_generator.py

```
@@ -1,4 +1,40 @@
+
+        command = [
+            "pandoc",
+            str(html_path),
+            "-o",
+            str(pdf_path),
+            "--pdf-engine=wkhtmltopdf",
+            "--metadata",
+            "title=CT report",
+        ]
+
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)